pkg/registry/lock/registry.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Can you please make it consistent with other resources and move this file to pkg/registry/lock/etcd/etcd.go